pico/api/src/test/java/io/helidon/pico/testsubjects/AbstractPicoServices.java

```diff
@@ -48,11 +46,6 @@ public Services services() {
         return null;
```

Comment:
You cannot return a `null` from public API.

Reply:
```java
abstract class AbstractPicoServices implements PicoServices {
```

Not public either.